app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,7 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.HomeAssistant"
         android:usesCleartextTraffic="true"
```

Comment:
Do we still need this?

Reply:
No, it was defaulted to false in target API 28 but since we override it here in the network security config we should be ok because that works since API level 24.